frontends/rioterm/src/context/mod.rs

```diff
@@ -570,6 +570,32 @@ impl<T: EventListener + Clone + std::marker::Send + 'static> ContextManager<T> {
         self.current_route = self.current().route_id;
     }
 
+    #[inline]
+    pub fn switch_to_next_split_or_tab(&mut self) {
```

Comment:
everything looks good, could you just add one test for next and prev so we make sure it always work 🙏

Reply:
Done. Thanks for the patience.